openlibrary/core/vendors.py

```diff
@@ -276,10 +277,28 @@ def get_betterworldbooks_metadata(isbn):
     isbn = normalize_isbn(isbn)
     try:
         if isbn:
-            return _get_betterworldbooks_metadata(isbn)
+            metadata = _get_betterworldbooks_metadata(isbn)
+            if not metadata.get('price') and thirdparty:
+                return _get_betterworldbooks_thirdparty_metadata(isbn)
+            return metadata
     except Exception:
         return {}
 
+def _get_betterworldbooks_thirdparty_metadata(isbn):
+    if isbn:
+        url = 'https://www.betterworldbooks.com/product/detail/-%s' % isbn
+        try:
+            r = requests.get(url)
+            results = [{
+                'url': BWB_AFFILIATE_LINK % (isbn),
+                'qlt': r[0].lower(),
+                'price': '$%s (%s)' % (r[1], r[0].lower()),
+                'price_amt': r[1],
+            } for r in re.findall('data-condition=\"(New|Used).*data-price=\"([0-9.]+)\"', r.content)]
```

Comment:
Don't need to escape `"`
```suggestion
            } for r in re.findall('data-condition="(New|Used).*data-price="([0-9.]+)"', r.content)]
```

Reply:
Done